packages/framework/fluid-static/src/fluidContainer.ts

```diff
@@ -37,6 +38,10 @@ export class FluidContainer extends TypedEventEmitter<IFluidContainerEvents> imp
         container.on("disconnected", this.disconnectedHandler);
     }
 
+    public get id() {
```

Comment:
Do we want to expose the container ID on the public API for the `FluidContainer`? I.e. is this something that we are only using to get the ID after creation? Or would we also expect developers to use this to fetch container IDs after a `getContainer` call?

Reply:
Getting an ID of an object is healthy, it can be used in multiple scenarios, such as logging or troubleshooting.

Reply:
`id` on Container will be removed from the public interfaces. This is tracked here - https://github.com/microsoft/FluidFramework/issues/2903
We are currently working on cleaning this up and I would rather not add more usages :-).

Reply:
There is an issue indicating we want to retiring Container.id #2903. 

I believe that we don't have a good definitions and requirement what contain.id means.  Currently it is just the id that gets populated by the url resolver.

If we want to to expose container.id, then we will need spell out the definition, requirement and usage, and make sure it is consistant and work across all the url resolve/drivers.

Reply:
@agarwal-navin @curtisman I can use a workaround in this iteration, but we still need a solution for retrieving the container id, aka `documentId`, generated by the backend
```ts
        const documentId = await ordererRestWrapper.post<string>(
            `/documents/${tenantId}`,
            {
                id,
                summary: convertSummaryToCreateNewSummary(appSummary),
                sequenceNumber: documentAttributes.sequenceNumber,
                values: quorumValues,
            },
        );
```
Currently, the r11s driver keeps it buried in the document service. Do we have any plans for refactoring it to surface the new container id?

Reply:
Reverted for now. Will address it the next PR.